admin_manual/configuration/server/harden_server.rst

```diff
@@ -128,13 +128,6 @@ If you have subdomains not accessible via HTTPS, remove ``includeSubDomains``.
 
 .. note:: This requires the ``mod_headers`` extension in Apache.
 
-When using NGINX as a Web server an example is already included in the
-:doc:`../installation/nginx_examples`::
-
-  #add_header Strict-Transport-Security "max-age=15552000; includeSubDomains";
-
-You need to remove the ``#`` and reload NGINX to enable this change.
-
```

Comment:
I'm not completely sure whether this should have been included in this PR. Likely it would have been a better fit in a separate one. That said, it was an easy thing to do while I was working in the file.